src/component/toolbox/feature/DataView.ts

```diff
@@ -363,7 +364,7 @@ class DataView extends ToolboxFeature<ToolboxDataViewFeatureOption> {
             textarea.readOnly = model.get('readOnly');
             const style = textarea.style;
             // eslint-disable-next-line max-len
-            style.cssText = 'width:100%;height:100%;font-family:monospace;font-size:14px;line-height:1.6rem;resize:none';
+            style.cssText = 'width:100%;height:100%;font-family:monospace;font-size:14px;line-height:1.6rem;resize:none;vertical-align:middle;box-sizing:border-box;outline:none';
```

Comment:
I'm not sure if this `vertical-align:middle;` is necessary.

Reply:
This is to remove the expected gap to avoid the scrollbar since the default `display` of `textarea` is `inline-block`. Of course, we can also set `display` as `block`.

Reply:
Didn't aware the default display of textarea is `inline-block`. Perhaps setting `display` to `block` will be better?

Reply:
Sure.